go.mod

```
@@ -0,0 +1,58 @@
+module github.com/DropsWeb/todo-go
```

Судя по названию репозитория должно быть:
```suggestion
module github.com/DropsWeb/todogo
```